crates/relayer/src/chain/cosmos.rs

```diff
@@ -296,27 +296,27 @@ impl CosmosSdkChain {
             ));
         }
 
-        // Query /genesis RPC endpoint to retrieve the `max_expected_time_per_block` value to use as `max_block_time`.
-        // If it is not found, keep the configured `max_block_time`.
-        match self.block_on(self.rpc_client.genesis::<GenesisAppState>()) {
-            Ok(genesis_reponse) => {
-                let old_max_block_time = self.config.max_block_time;
-                let new_max_block_time =
-                    Duration::from_nanos(genesis_reponse.app_state.max_expected_time_per_block());
-
-                if old_max_block_time.as_secs() != new_max_block_time.as_secs() {
-                    self.config.max_block_time = new_max_block_time;
-
-                    info!(
-                        "Updated `max_block_time` using /genesis endpoint. Old value: `{}s`, new value: `{}s`",
-                        old_max_block_time.as_secs(),
+        // Query Connection Params with gRPC endpoint to retrieve the `max_expected_time_per_block` value and verify the
+        // configured `max_block_time`.
+        // If it is not found, the verification for the configured `max_block_time` is skipped.
+        match self.block_on(query_connection_params(&self.grpc_addr)) {
+            Ok(params) => {
+                warn!("raw max block time: {}", params.max_expected_time_per_block);
```

Comment:
```suggestion
                info!("chain queried max block time: {}", params.max_expected_time_per_block);
```

Reply:
Oh I forgot to remove this log, I don't need it's necessary to keep it. Or maybe putting it as `debug`, what do you think?

Reply:
imo we should have a log, debug is also fine.

Reply:
I added a debug log with the unprocessed queried value (before creating a `Duration` using the value) here 7d736c04be9fcadb74482ca7f2674a86be34a6c6